substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -1417,9 +1417,70 @@ where
 		// transactions with those hashes from the retracted blocks.
 		let mut pruned_log = HashSet::<ExtrinsicHash<ChainApi>>::new();
 
+		// Create a map from (retracted blocks + common block)'s extrinsics to their `provides`
+		// tags.
+		let inactive_views_xts_to_tags = Arc::new(
```

Comment:
```suggestion
		let known_provides_tags = Arc::new(
```

or maybe `..provided..` sounds better?

Reply:
Yes, updated here: [8de3686](https://github.com/paritytech/polkadot-sdk/pull/7980/commits/8de368694385db28c903323ad9779fc1b87b9dda).